client/components/NetworkForm.vue

```diff
@@ -167,6 +167,16 @@
 					maxlength="300"
 				/>
 			</div>
+			<div class="connect-row">
+				<label for="connect:leaveMessage">Away message</label>
+				<input
+					id="connect:leaveMessage"
+					v-model="defaults.leaveMessage"
+					class="input"
+					name="leaveMessage"
+					placeholder="Smell ya later"
```

Comment:
I agree with @Jay2k1, can we adjust this to be something more serious than Smell ya later, like `The Lounge - https://thelounge.chat` (which is the default global leaveMessage)